server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
The other way to get to having a single pref, would be for tokenserver to export a similar document that points back to sync.  i.e. the user sets *just* the location of the "sync server" to the tokenserver endpoint.  Firefox does discovery on that endpoint to determine the corresponding accounts server, then does discovery on the accounts server to find all the endpoints it needs.

(I think this is how I originally imagined it happening several years ago, but it always seemed like a bit much client complexity)